content/guides/models/track/log/customize-logging-axes.md

```diff
@@ -6,51 +6,90 @@ menu:
 title: Customize log axes
 ---
 
-Use `define_metric` to set a **custom x axis**.Custom x-axes are useful in contexts where you need to log to different time steps in the past during training, asynchronously. For example, this can be useful in RL where you may track the per-episode reward and a per-step reward.
+Set a custom x-axis when you log metrics to W&B. By default, W&B logs metrics as *steps*. Each step corresponds to a `wandb.log()` API call. 
 
-[Try `define_metric` live in Google Colab →](http://wandb.me/define-metric-colab)
+For example, suppose you have a `for` loop that loops 10 times (see the following code block). In each `for` loop, you log a metric called `validation_loss` with `wandb.log()`. W&B increments the step number by 1 each time you log that metric. 
 
-### Customize axes
+```python
+import wandb
+
+with wandb.init() as run:
+  # range function creates a sequence of numbers from 0 to 9
+  for i in range(10):
+    log_dict = {
+        "validation_loss": 1/(i+1)   
+    }
+    run.log(log_dict)
+```
+
+If you navigate to your project's workspace, you can see that the `validation_loss` metric is plotted against the step x-axis. Each step in the for loop monotonically increases by 1, so the x-axis shows the step numbers 0, 1, 2, ..., 9. (The Python `range()` function creates a sequence of numbers from 0 to 9.)
```

Comment:
```suggestion
In the project's workspace, the `validation_loss` metric is plotted against the `step` x-axis, which increments by 1 each time `run.log()` is called.
```
You already have a comment in the script that explains `range()` if it's not assumed knowledge for Python developers.